src/components/InventoryTable/InventoryTable.js

```diff
@@ -146,6 +149,18 @@ const InventoryTable = forwardRef(
         abortOnUnmount && controller.current.abort();
       };
     }, []);
+    const hasLoadEntitiesError =
+      error?.status === 404 &&
+      error?.type === 'LOAD_ENTITIES' &&
```

Comment:
I think you can import a constant with the LOAD_ENTITIES value from src/store/action-types.js

Reply:
Good idea, thank you!